mozilla/activity-stream
# Bug 1568067 - Adding in more to DS devtools

To test:
1. Enable DS devtools with this pref: browser.newtabpage.activity-stream.asrouter.devtoolsEnabled set to true
2. Open a new tab
3. Click the little wrench in the top left.
4. In the top right open the Discovery Stream tab.
5. Test that you can reset the pref to default
6. Test that you can clear cache, this includes frequency capping.
7. Test that you can enable and disable all the checkboxes and they do what's expected.
